src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -47,7 +47,7 @@ public boolean doMonitor(ServiceEmitter emitter)
       final String kind = entry.getKey();
```

Comment:
any changes in this PR will require updates to the docs in https://github.com/druid-io/druid/pull/1208